package.json

```diff
@@ -102,6 +102,8 @@
     "@types/node": "^18.7.16",
     "@types/react": "^18.2.15",
     "@types/react-dom": "^18.2.7",
+    "@types/react-modal": "^3.16.2",
```

Comment:
Why do we need these in dev dependencies?

Reply:
These packages contain type definitions for plain JS packages that don't provide type definitions themselves.
https://definitelytyped.org/

TypeScript automatically uses them.
https://www.typescriptlang.org/tsconfig/#typeRoots